src/smpi/instr_smpi.cpp

```diff
@@ -195,19 +204,21 @@ void TRACE_smpi_init(int rank)
   char str[INSTR_DEFAULT_STR_SIZE];
   smpi_container(rank, str, INSTR_DEFAULT_STR_SIZE);
 
-  container_t father;
+  container_t father, me;
   if (TRACE_smpi_is_grouped()){
     father = PJ_container_get (SIMIX_host_self_get_name());
   }else{
     father = PJ_container_get_root ();
   }
   xbt_assert(father!=nullptr,
       "Could not find a parent for mpi rank %s at function %s", str, __FUNCTION__);
+  me = PJ_container_new(str, INSTR_SMPI, father);
```

Comment:
Please prefer declaring and affecting value on the same line, as in "container_t me = PJ_container...."

I know it's not the case everywhere in the code base, but that's the kind of fixes I do all the time so I'd prefer to have it already done here. Thanks.